doc/ansible/dev/testing_guide.md

```diff
@@ -231,13 +206,11 @@ We'll be adding the task to `roles/example/tasks/main.yml`, which should now loo
         hello: world
 ```
 
-
-
 ### Adding a test
 
 Now that our Operator actually does some work, we can add a corresponding assert to `molecule/cluster/verify.yml`.
-We'll also add a debug message so that we can see what the ConfigMap looks like.
-The file should now look like this:
+
+We'll also add a debug message so that we can see what the ConfigMap looks like. The file should now look like this:
```

Comment:
This word wrapping doesn't change how things are rendered, but does make diffs a lot easier to see (smaller changes). Why undo that?

Reply:
The doc was with spaces in the line which would not render appropriated according to the layout/css applied.

Reply:
Interesting. Could you show a screenshot, this looks correct on current master:

![image](https://user-images.githubusercontent.com/1028657/79159541-45f5c800-7da6-11ea-8c65-d259c61d3e30.png)

Reply:
You are right here. I reverted this change.